client/views/modalCloseButton.jade

```diff
@@ -0,0 +1,4 @@
+template(name="modalCloseButton")
+  button.close(type="button" data-dismiss="modal" aria-label="Close")
+    span(aria-hidden="true")
+      i.fa.fa-times
```

Comment:
This may have been picked-up too soon. In #90 I introduced an element and class ([here](https://github.com/ecohealthalliance/eidr-connect/blob/cf73099c699554bbbfc4fe16b626aa01178e4a61/client/views/userEvent.jade#L137)). Maybe once that's merged we can combine these ideas?